sources/iTermAdvancedSettingsModel.m

```diff
@@ -151,4 +151,11 @@ + (NSString *)name { \
 
 DEFINE_BOOL(noSyncTipsDisabled, NO, @"Tip of the Day: Disable the Tip of the Day?");
 
+#pragma mark - Badge
+DEFINE_STRING(badgeFont, @"Helvetica", @"Badge: Font to use for the badge.");
+DEFINE_BOOL(badgeFontIsBold, YES, @"Badge: Make the badge font bold.");
+DEFINE_FLOAT(badgeMaxWidthPercent, 0.5, @"Badge: Maximum width of the badge\nAs a percentage of the width of the terminal, between 1% and 100%.");
```

Comment:
It's a fraction, not a percentage. The acceptable values are between 0 and 1. Same for badgeMaxHeightPercent.